api/store/mongo/store.go

```diff
@@ -193,7 +193,8 @@ func (s *Store) DeleteDevice(ctx context.Context, uid models.UID) error {
 		return err
 	}
 
-	return nil
+	_, err := s.db.Collection("connected_devices").DeleteMany(ctx, bson.M{"uid": uid})
```

Comment:
Shouldn't this to be `DeleteOne`?

Reply:
No, because there can be more than one document.